spec/hyperYieldSpec.js

```diff
@@ -0,0 +1,20 @@
+describe('hyper-yield tag', function() {
+  var $compile,
+      $rootScope;
+
+  beforeEach(module('hyperContentFor'));
+
+  beforeEach(inject(function(_$compile_, _$rootScope_){
+    $compile = _$compile_;
+    $rootScope = _$rootScope_;
+  }));
+
+  it('Replace with content from matching hyper-content', function() {
```

Comment:
I personally prefer writing these so that they read well together with the “if”. In other words, like this:

```javascript
it("replaces with content from matching hyper-content", function () { ... });
```

What do you think? Also, this should totally be mentioned in our [playbook]!

[playbook]: https://github.com/hyperoslo/javascript-playbook